src/couch_peruser/src/couch_peruser.app.src

```diff
@@ -14,5 +14,8 @@
     {description, "couch_peruser - maintains per-user databases in CouchDB"},
     {vsn, git},
     {registered, []},
```

Comment:
Can add the registered modules if you want. I think just `couch_peruser_sup` and `couch_peruser`?